docs/experimental.md

```diff
@@ -128,3 +128,38 @@ Set the location of your manually downloaded database by following the instructi
 ### Limitations
 
 1. Commit level scanning is not supported. 
+
+## License scanning
+
+OSV-Scanner supports license checking as an experimental feature. The data comes from the [deps.dev API](https://docs.deps.dev/api/v3alpha/).
```

Comment:
I'm not sure what's better, https://docs.deps.dev/api/ or https://docs.deps.dev/api/v3alpha/. I know that /api redirects to the latest version of the api docs (although in this case the only version is v3alpha). We do use v3alpha in this feature at the moment, so I think it's fine.

Reply:
Let's do https://docs.deps.dev/api/. We are not linking to the specific part of the API we are using, but just pointing people to a resource where they can learn more if they would like.